client/components/Products.js

```diff
@@ -8,35 +8,60 @@ import history from '../history'
 /**
  * COMPONENT
  */
-export const Products = (props) => {
-  const {products, categories} = props
-
-  return (
-    <div>
-      <h1>All Products Page</h1>
-      <div className="categories">
-        {categories.map( category => (
-          <h3 key={category.id}>{category.name}</h3>
-        ))}
-      </div>
-      {products.map( product => {
-        return (
-        <div key = {product.id}>
-          <Link to={`/products/${product.id}`}>
-            <h3>{product.name}</h3>
-            <img src = {product.image} />
-          </Link>
-          <h3>{product.price}</h3>
-          <p>{product.description}</p>
-          <h3>{product.stock} </h3>
-          <h3>{product.isAvailable} </h3>
-          <h3>{product.category}</h3>
-          <h3>{product.reviews}</h3>
+export class Products extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      currentCategory: 'all'
+    }
+    this.updateCategory = this.updateCategory.bind(this);
+  }
+
+
+  updateCategory = (category) => {
+    console.log('updating state')
```

Comment:
do you want to keep the console.logs and then in the end, we'll review and remove all console logs